### DISCUSSION TITLE
wsl: DNS Tunneling is not supported

### DISCUSSION THREAD

**Discussion by blakeduffey** [microsoft/WSL#10502] [General] [answered]
What version of Windows is required to support wsl DNS Tunneling?  (I'm assuming that is my issue)

**Comment by creizlein** [comment, marked as answer]
According to [this post](https://devblogs.microsoft.com/commandline/windows-subsystem-for-linux-september-2023-update/)
---
Please note this feature is currently only available to Windows Insiders canary and Release Preview Channel with the latest Windows 11, version 22H2 update [here](https://blogs.windows.com/windows-insider/2023/09/14/releasing-windows-11-build-22621-2359-to-the-release-preview-channel/). You can get access now by [joining the Windows Insider Program](https://support.microsoft.com/en-us/windows/join-the-windows-insider-program-and-manage-insider-settings-ef20bb3d-40f4-20cc-ba3c-a72c844b563c) and choosing to opt in your device into the Release Preview Channel.

**Reply by ltalirz** [reply]
Where can I find information on when this feature will be released to users who are not registered for the Windows Insider program?

The [docs on Windows 11 22H2](https://learn.microsoft.com/en-us/windows/release-health/status-windows-11-22h2) read as if it was already available to all users.

**Comment by blakeduffey** [comment]
Thanks much.  This release works on my current version of Windows 10 
Windows version: 10.0.19044.3448

But WSL gets all cranky with any of the experimental flags enabled in the config file.
